goldens/public-api/angular_devkit/build_angular/index.md

```diff
@@ -4,6 +4,8 @@
 
 ```ts
 
+/// <reference types="@types/node/http" />
```

Comment:
We saw similar references being added in `@angular/localize` as well https://github.com/angular/angular/pull/52572#discussion_r1385665869